_docs/install/docker/index.md

```diff
@@ -7,31 +7,34 @@ module: install/docker/index
 
 # Overview
 
-StorageOS can be installed in two ways.
+StorageOS can be installed in one of two ways.
 
-1. Kubernetes and Docker 1.10 - 1.13 users should use the [container install]({%
-link _docs/install/docker/plugin.md %}).
+1. For Kubernetes and Docker 1.10+, use the [container install]({%
+link _docs/install/docker/container.md %}).
 
-2. All other users (eg. Docker Swarm, testing) should use the [volume plugin]({%
+2. For Docker 1.13+ testing, use the [volume plugin]({%
 link _docs/install/docker/plugin.md %}).
 
+>**Do not install both the node and the plugin**. The plugin is harder to debug, so if in doubt, use the container install.
+
 With either approach, you will need to install StorageOS on every node in a
 cluster.
 
-Ensure you read [the prerequisites]({% link _docs/install/prerequisites/index.md %})
-before installing StorageOS.
+Ensure [the prerequisites]({% link _docs/install/prerequisites/index.md %}) are
+fulfilled before installing StorageOS.
 
 ## Environment variables
 
-For most environments, the default settings should work. If Consul is not
-running locally on the node, you will need to set `KV_ADDR`.
+For most environments, the default settings should work.
 
 * `HOSTNAME`: Hostname of the Docker node, only if you wish to override it.
 * `ADVERTISE_IP`: IP address of the Docker node, for incoming connections.  Defaults to first non-loopback address.
 * `USERNAME`: Username to authenticate to the API with.  Defaults to `storageos`.
```

Comment:
Missing?  (descriptions copied from config.json - may not be ideal)
CLUSTER_ID: Cluster ID for the node to join an existing cluster previously created through 'storageos cluster create' command
INITIAL_CLUSTER: Static list of pre-existing cluster. Supplied in the form: 'storageos-1=NODE1_URL,storageos-2=NODE2_URL,storageos-3=NODE3_URL